dev_algorithms/wavelengths/template_files/Keck_DEIMOS/1200B/README

```diff
@@ -0,0 +1,10 @@
+*.dat files
+IRAF outputs from Carlos (WMKO)
```

Comment:
Should we convert the template to vacuum?

Reply:
they were

Reply:
maybe make that clear in the README. .